projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -1080,9 +1080,6 @@ export function clickAddToCartBtn(): void {
     .then(() => {
       cy.location('pathname').should('contain', 'cartEntry/entityKey/');
       checkUpdatingMessageNotDisplayed();
-      checkSuccessMessageNotDisplayed();
```

Comment:
I think we can re-enable this check after Lukas' changes, right?

Reply:
We don't have this checks in the develop branch. Therefore I think we can remove them.

Reply:
Ah I see. Agree